### ISSUE TITLE
UX: show history visibility in title

### ISSUE THREAD

**Issue by rubo77** [vector-im/riot-web#6821]
there are 4 options for "Who can read history?"

```
   (x) Anyone
   ( ) Members only (since the room began)
   ( ) Members only (since they were invited)
   ( ) Members only (since they joined)
```

This is an important setting for users in the room, so it should be visible in every   room on the top.

You    should know, that what you say in the room is only readeable for the users currently in the room or also later by new members.

**Comment by lampholder** [comment]
I agree with this (and don't think what I'm about to say next should block pursuing this) but I think there are a few additional categories of information it would be useful to make sure are super visible to users in a room, such as:
- is this room bridged
- are there widgets in this room which have permission to read room messages
- are there bots in the room
- is the room encrypted
- does this room support federation
- how many servers are federated
- etc.

**Comment by Mikaela** [comment]
👍🏻  I am not sure if title is the right place for this, but I agree that the information should be shown more clearly.

Matrix is at the point where there are rooms for ~everything including sensitive subjects or peer groups around them and as an user it's not pleasant to discover that the admin had set room history visibility as public in hopes of getting more users in such a room.
